doc/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx
 sphinx_rtd_theme>=1.3
 sphinx-click
+sphinx-lint
```

Comment:
In this CI, we also need a dependency to "sphinx-intl" (for released) or `.` (for HEAD).

It is used at:
https://github.com/rffontenelle/sphinx-intl/blob/0b90c321291f593a4aaeb9067b16f67e4b9a862b/doc/conf.py#L3

I tried this CI by `act -j doc`, it raise a exception as bellow:
```
[Docs/doc] ⭐ Run Main Check links
[Docs/doc]   🐳  docker exec cmd=[bash -e /var/run/act/workflow/5] user= workdir=
| make: Entering directory '/workspaces/sphinx-intl/doc'
| Running Sphinx v8.1.3
| 
| Configuration error:
| There is a programmable error in your configuration file:
| 
| Traceback (most recent call last):
|   File "/opt/hostedtoolcache/Python/3.13.0/x64/lib/python3.13/site-packages/sphinx/config.py", line 529, in eval_config_file
|     exec(code, namespace)  # NoQA: S102
|     ~~~~^^^^^^^^^^^^^^^^^
|   File "/workspaces/sphinx-intl/doc/conf.py", line 3, in <module>
|     from sphinx_intl import __version__
| ModuleNotFoundError: No module named 'sphinx_intl'
| 
| make: *** [Makefile:20: linkcheck] Error 2
| make: Leaving directory '/workspaces/sphinx-intl/doc'
[Docs/doc]   ❌  Failure - Main Check links
[Docs/doc] exitcode '2': failure
[Docs/doc] 🏁  Job failed
```

Reply:
So which one should I choose? I'm leaning to `.` (HEAD) as this is mostly used for development, but don't know how this will affect anyone building a future released version from source (is it possible to load incorrect dependencies?)

Reply:
I prefer to use HEAD.
The testing in CI for doc is done in HEAD, and I think it is natural that the source code referenced by the doc should also be in HEAD at that time.

Reply:
```suggestion
sphinx-lint
.
```